v2/pkg/genruntime/kubernetes_resource.go

```diff
@@ -6,11 +6,9 @@
 package genruntime
 
 import (
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime/conditions"
```

Comment:
minor: Fix ordering, this should be sorted to the bottom.

I think there's an option in goland to do this automatically for you.

Reply:
ah! this is what goland did it for me 😆. I'll make it align to our coding convention.